app/assets/javascripts/directives/validation/reset_validation_status.js

```diff
@@ -15,14 +15,14 @@ ManageIQ.angular.app.directive('resetValidationStatus', ['$rootScope', function(
 }]);
 
 var adjustValidationStatus = function(value, scope, ctrl, attrs, rootScope) {
-  if(scope.checkAuthentication === true &&
+  if (scope.checkAuthentication === true &&
      scope.postValidationModel !== undefined &&
      scope.postValidationModel[attrs.prefix] !== undefined) {
     var modelPostValidationObject = angular.copy(scope.postValidationModel[attrs.prefix]);
     delete modelPostValidationObject[ctrl.$name];
 
     var modelObject = angular.copy(scope[scope.model]);
-    if(modelObject[ctrl.$name] !== undefined) {
+    if (modelObject[ctrl.$name]) {
```

Comment:
This is probably wrong.. we're deleting the current field from the model object so that we can check for match without it.

This will keep empty strings undeleted, which may change whether the comparison succeeds.